addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14663,7 +14663,55 @@ msgctxt "#24151"
 msgid "Add-on migration in progress - please wait..."
 msgstr ""
 
-#empty strings from id 24152 to 24990
+#. Localized addon database name for splash screen
+#: xbmc/addons/AddonDatabase.h
+msgctxt "#24152"
+msgid "Addon"
```

Comment:
The other view is that storing the same text over and over is wasteful.
But is it a translation issue e.g. in some languages the object needs to go else where in the phrase?